website/components/Heading.tsx

```diff
@@ -7,8 +7,8 @@ export type HeadingProps = JSX.IntrinsicElements['h1'];
 
 export function Heading({ children, className, ...props }: HeadingProps) {
   return (
-    <h1 {...props} className={classNames(styles.heading, className)}>
+    <h2 {...props} className={classNames(styles.heading, className)}>
```

Comment:
Can you make sure whether the ordering of headings is correct with this change? I can't remember the tool that @wadamek65 recommended some time ago.

Reply:
You can use this plugin: https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi

Reply:
Thanks for the nice plugin. I checked it and there was only one incorrect ordering - the one from [this comment](https://github.com/vazco/uniforms/pull/1117#discussion_r884553295).